packages/drivers/odsp-driver/src/test/epochTests.spec.ts

```diff
@@ -41,13 +41,9 @@ describe("Tests for Epoch Tracker", () => {
         cache.put(cacheEntry1, { value: "val1", fluidEpoch: "epoch1", version: "0.1" }, 0);
         cache.put(cacheEntry2, { value: "val2", fluidEpoch: "epoch2", version: "0.1" }, 0);
         // This will set the initial epoch value in epoch tracker.
-        await mockFetch({}, async () => {
```

Comment:
@jatgarg, please take a closer look here. I removed in couple placed wrapping with mockFetch, because we are not doing fetches - we are getting stuff from cache. Extra wrappers were not adding anything and where making code less readable, IMHO